packages/feedback-spa/src/app/app.service.ts

```diff
@@ -7,7 +7,7 @@ import { GraphQLModule } from './apollo.config';
 export class AppService extends GraphQLModule {
   constructor(
     private apollo: Apollo,
-    private httpLink: HttpLink) 
+    private httpLink: HttpLink)
     {
     super(apollo, httpLink);
   }
```

Comment:
Fix the indentation. And also, can you fix your local ts lint. I see a lot of linting issues in your code. Even in other places.

Reply:
I am using 1 tab to the code that is inside any function or any constructor. can you please tell what rule we are following the code in our project.

Reply:
Check the tslint.json file in the project root, and your package root. All the style rules are defined in there. And your IDE should be able to pick it and lint your code according to those rules.
Maybe make sure yoou have installed the tslint extension for VS Code, and update your VS Code to the latest version, if it's not already.